json_serializable/test/test_files/kitchen_sink.g.dart

```diff
@@ -38,16 +38,12 @@ KitchenSink _$KitchenSinkFromJson(Map<String, dynamic> json) => new KitchenSink(
       ? null
       : new Map<String, DateTime>.fromIterables(
           (json['stringDateTimeMap'] as Map<String, dynamic>).keys,
-          (json['stringDateTimeMap'] as Map)
-              .values
-              .map((e) => e == null ? null : DateTime.parse(e as String)))
+          (json['stringDateTimeMap'] as Map).values.map(
+              (dynamic e) => e == null ? null : DateTime.parse(e as String)))
   ..crazyComplex = (json['crazyComplex'] as List)
-      ?.map((e) => e == null
+      ?.map((dynamic e) => e == null
           ? null
-          : new Map<String, Map<String, List<List<DateTime>>>>.fromIterables(
-              (e as Map<String, dynamic>).keys,
-              (e as Map).values.map((e) =>
-                  e == null ? null : new Map<String, List<List<DateTime>>>.fromIterables((e as Map<String, dynamic>).keys, (e as Map).values.map((e) => (e as List)?.map((e) => (e as List)?.map((e) => e == null ? null : DateTime.parse(e as String))?.toList())?.toList())))))
+          : new Map<String, Map<String, List<List<DateTime>>>>.fromIterables((e as Map<String, dynamic>).keys, (e as Map).values.map((dynamic e) => e == null ? null : new Map<String, List<List<DateTime>>>.fromIterables((e as Map<String, dynamic>).keys, (e as Map).values.map((dynamic e) => (e as List)?.map((dynamic e) => (e as List)?.map((dynamic e) => e == null ? null : DateTime.parse(e as String))?.toList())?.toList())))))
```

Comment:
wow lol... presumably dartfmt is just giving up here?

Reply:
Yup!